docs/content/user-guides/cert-manager.md

```diff
@@ -0,0 +1,155 @@
+---
+title: "Traefik Cert Manager Integration"
+description: "Learn how to configure Traefik Proxy to use Cert Manager for certificate on your routers. Read the technical documentation."
+---
+
+# Cert Manager
+
+Provision TLS Certificate for Traefik Proxy with Cert Manager on Kubernetes
+{: .subtitle }
+
+You can configure Traefik Proxy to use Certificates of Cert Manager on Kubernetes.
+
+## Checklist
+
+To obtain a certificate from Cert Manager, you'll need to:
+
+1. Deploy Cert Manager
+2. Configure an Issuer (or a ClusterIssuer)
+3. Create a Certificate with this Issuer
+4. Deploy Traefik Proxy
+5. Use the certificate in an Ingress / IngressRoute / HTTPRoute
+
+## Configuration Example with ACME and HTTP challenge
+
+!!! example "ACME issuer on http challenge"
+
+    ```yaml
+    apiVersion: cert-manager.io/v1
+    kind: Issuer
+    metadata:
+      name: acme
+
+    spec:
+      acme:
+        # Production server is on https://acme-v02.api.letsencrypt.org/directory
+        # Use staging by default.
+        server: https://acme-staging-v02.api.letsencrypt.org/directory
+        privateKeySecretRef:
+          name: acme
+        solvers:
+          - http01:
+              ingress:
+                ingressClassName: traefik
+    ```
+
+!!! example "Certificate with this Issuer"
+
+    ```yaml
+    apiVersion: cert-manager.io/v1
+    kind: Certificate
+    metadata:
+      name: whoami
+      namespace: traefik
+    spec:
+      secretName: domain-tls        # <===  Name of secret where the generated certificate will be stored
+      dnsNames:
+        - "domain.example.com"
+      issuerRef:
+        name: acme
+        kind: Issuer
+    ```
+
+!!! example "Route with this Certificate"
```

Comment:
Here I would have created a new section, something like how to use the certificate generated above, wdyt?